packages/language-server/test/plugins/typescript/features/SemanticTokensProvider.test.ts

```diff
@@ -94,6 +94,20 @@ describe('TypeScript Plugin#SemanticTokenProvider', () => {
 				type: TokenType.variable,
 				modifiers: [TokenModifier.declaration, TokenModifier.readonly],
 			},
+			{
+				line: 10,
+				character: 19,
+				length: 'Astro'.length,
+				type: TokenType.type,
+				modifiers: [TokenModifier.readonly],
+			},
+			{
+				line: 10,
+				character: 25,
+				length: 'props'.length,
+				type: TokenType.property,
+				modifiers: [],
+			},
```

Comment:
Previously our tests couldn't get the types properly, thus it wouldn't get the proper semantic tokens :sweat_smile: